### PR TITLE
Add doom, ticketmaster, superfreezz & shortcut

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

**Review by Chefski — CHANGES_REQUESTED** [review]
`mighty_doom` can fit the 160x160px content rather than 144x144px
image

Also sort the links in alphabetical order dude. Everyone does it. If you don't want to do it manually then do it semi automatically with the `icontool.py`.

**Comment by peterge1998** [comment]
Rescale of doom done

**Comment by peterge1998** [comment]
I added a stupid ssort.sh script, which just does all i want to do automatically. now just add the lines to the appfilter.content and run sort - no manual sorting is needed at all...

**Review by daywalk3r666 — CHANGES_REQUESTED** [review]
The doom icon has a stroke over 13px.

File: svgs/mighty_doom.svg
Comment (daywalk3r666) [inline svgs/mighty_doom.svg:1]: The icon has a stroke over 13px

**Review by daywalk3r666 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**